fhir-server/liberty-config/configDropins/disabled/cloud.xml

```diff
@@ -1,7 +1,7 @@
 <server>
     <!-- webApp security is MUST be provided via a gateway -->
```

Comment:
not related to your change at all, but i just spotted this typo that i must have introduced earlier:
```suggestion
    <!-- webApp security MUST be provided via a gateway -->
```

Reply:
Fixing in my upcoming pr